### PR TITLE
Make sure link color actually appears

### PR DESCRIPTION
**Pull request by bernhardmgruber**

### CODE REVIEW

File: talk/setup.tex [hunk 1]
```diff
@@ -275,5 +275,7 @@
 }
 
 \hypersetup{
+  colorlinks=true,
+  allcolors=.,
   urlcolor={\color{blue!50!white}}
```
Comment (hageboeck) [inline talk/setup.tex:280]: Like this:
```suggestion
  urlcolor={blue!50!white}
```
The `\color` is prepended also by hyperref

Comment (bernhardmgruber) [reply talk/setup.tex]: Thx for spotting this!

**Comment by hageboeck** [comment]
How about giving it a bit more blue, maybe `blue!80!white`?

**Review by hageboeck — APPROVED** [review]

**Comment by bernhardmgruber** [comment]
> How about giving it a bit more blue, maybe blue!80!white?

I really don't care :D
Done.

**Pull request merged**